pkgs/tools/text/papeer/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "papeer";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "lapwat";
+    repo = pname;
```

Comment:
Don't parameterize pname.

Reply:
Oh fuck, the "shut up" ghost case scares me again. I will refrain for now.

Reply:
I don't know about the shut-up-ghost but people tend to copy-paste package expressions and I like the way this breaks when src hasn't been updated.